src/components/Dashboard/index.js

```diff
@@ -108,31 +107,6 @@ class Dashboard extends Component {
     })
   }
 
-  renderClosingMarkets(markets) {
-    return markets.map((market) => {
-      const viewMarket = () => this.handleViewMarket(market)
-      const outcomeOptions = {
-        showOnlyTrendingOutcome: true,
-      }
-
-      return (
-        <div
-          className="dashboardMarket dashboardMarket--closing dashboardMarket--twoColumns"
-          key={market.address}
-          onClick={viewMarket}
-        >
-          <div className="dashboardMarket__leftCol">
-            <div className="value">{moment.utc(market.eventDescription.resolutionDate).fromNow()}</div>
-          </div>
-          <div className="dashboardMarket__rightCol">
-            <div className="dashboardMarket__title">{market.eventDescription.title}</div>
-            <Outcome market={market} opts={outcomeOptions} />
-          </div>
-        </div>
-      )
-    })
-  }
-
   renderMyHoldings(holdings) {
```

Comment:
Do not know if refactor should be done at this point, but I would extract MyHoldings to a private component and the reduces the complexity of lines 132-138